### PR TITLE
Adding command to export to `index.html`

### PR DESCRIPTION
**Pull request by beersweetpicklepanda**
Hey @LiveDuo,

I don't understand `next.js`, but I needed the project to have `index.html` that other web servers could run it. 

I found `next.js` can leverage the `export` command, and it seems to be working fine. Let me know your thoughts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,10 @@ npm run dev
 ```
 
 Deploy it to the cloud with [Vercel](https://vercel.com/new) ([Documentation](https://nextjs.org/docs/deployment)).
+
+If you need to deploy it with `index.html`, export it with:
```
Comment (LiveDuo) [inline README.md:34]: Export the project to static HTML with:

Comment (LiveDuo) [inline README.md]:
```suggestion
Export the project to static HTML with:
```

File: README.md [hunk 2]
```diff
@@ -30,3 +30,10 @@ npm run dev
...
+```sh
+npm run export
+```
+
+`index.html` will be exported in the `out` directory
```
Comment (LiveDuo) [inline README.md]:
```suggestion
This will export `index.html` to the `out` directory.
```

**Review by LiveDuo — CHANGES_REQUESTED** [review]
We can have `next export` command. Just some minor grammatical comments.

**Comment by beersweetpicklepanda** [comment]
Done, thanks for the correction! 👍

**Pull request merged**